lib/resources/example_json/r4_examples_questionnaire_response.rb

```diff
@@ -4,7 +4,77 @@ module Cerner
   module Resources
     R4_QUESTIONNAIRE_RESPONSE_ENTRY ||= {
       'resourceType': 'QuestionnaireResponse',
-      'id': 'SH-12508041'
+      'id': 'SH-12508041',
+      'questionnaire': 'https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041',
```

Comment:
fhir-ehr-code? i dont think thats the url ur calling?

Reply:
oop fixed: https://github.com/cerner/fhir.cerner.com/pull/669/commits/1f076c37e6dde09dd80de79ed2cbe13f4a4b8796

Reply:
let's just do fhir-ehr-code as the base all over